### DISCUSSION TITLE
Can't get my student information verified

### DISCUSSION THREAD

**Discussion by funnydevfacts** [community/community#139319] [GitHub Education]
### Hi everyone,

I cant get my student information approved. My student id card doesn't contain a date of validation so im trying to send some goverment approved paper but it is not working either. What should i do?

And also i live in the same city with my university so im not in campus right now. What can i provide regarding to that situation? 

Thanks

**Comment by davevad93** [comment]
Hi @funnydevfacts ,

The [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) state the following:

- ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.*** 

Which means that you can provide different types of documentation, for example, I was approved by uploading a screenshot of my last exam results, the most important thing is that in addition to the name and academic affiliation, there's a current or very recent date that proves your current student status; i.e. an enrollment sheet showing only the academic year wouldn't be enough.

For more information about rejections, take a look at the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).

**Comment by CinarKK9** [comment]
kanka takdir veya teşekkür belgesini çek bende olmuştu

**Reply by CinarKK9** [reply]
okul olarak da liseyi seç
